core/cli/timetravel.js

```diff
@@ -0,0 +1,62 @@
+const _ = require('lodash');
+const knex = require('../server/data/db/connection');
+const schema = require('../server/data/schema');
+const {DateTime} = require('luxon');
+const Command = require('./command');
+
+// we use logins as the basis for our offset
+const datum = {
+    table: 'members_login_events',
+    column: 'created_at'
+};
+
+module.exports = class TimeTravel extends Command {
+    async handle() {
+        const datumPoint = await knex(datum.table)
+            .max(datum.column, {as: datum.column})
+            .first();
+
+        const dateOffset = Math.floor(
```

Comment:
We should probably handle `members_login_events` being empty, otherwise I get:

```
...
[2022-05-16 17:32:48] INFO Timetravel will use an offset of +NaN days
...
```